ManatiCR/vitaminau
# VIT-28: Work in progress form.

# Task [VIT-28](https://manati.atlassian.net/browse/VIT-28?atlOrigin=eyJpIjoiM2EwNTg3ZjdhNjhmNDE5NzhhYjUyMTI1NDhhMDUyZGQiLCJwIjoiaiJ9)
Creation of the "Trabajo en Curso" form.

#Installation
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following url: 
  - http://vitaminau.docker/user
- Login as admin user:
  - User: admin
  - Pass: admin
- Go to the following url: 
  - http://vitaminau.docker/form/work-in-progress
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-28](https://manati.atlassian.net/browse/VIT-28?atlOrigin=eyJpIjoiM2EwNTg3ZjdhNjhmNDE5NzhhYjUyMTI1NDhhMDUyZGQiLCJwIjoiaiJ9)
- Fill all the fields of the form and press the save buttom
- Once you press it your information should be saved and the form should redirect you to a thank you page
- Save your draft with the button "Save Draft" and log in again in another browser or incognito, a "Se ha cargado el borrador." message should appear.
- To test the required fields, leave a blank, which has a red "*" and the same form will tell you that it has not been completed yet.


# Acceptance criteria
- Only authenticated users can access the form.
- Form can be saved as draft.
- When the required fields are empty, you should see a message, indicating that it must be filled
- I can complete the form and send it if I complete it correctly.
- The following fields are not required
  - PDF de programa del curso
  - Restricciones
  - ¿Algo que agregar?
  - PDF relevante
  - ¿Cómo calificaría este formulario?
  - ¿Qué mejoraría?
